### ISSUE TITLE
Bug - Can't find release plans and API Spec work item missing 

### ISSUE THREAD

**Issue by ladonnaq** [Azure/azure-sdk-tools#7148]
I am working with the Network Security Perimeter team.   I cannot find any of their release plans when I set the service context to their service - Network Security Perimeter and the API Spec work item is not in ADO.  

- Here is the Private Preview release plan link - https://github.com/Azure/azure-rest-api-specs/pull/21617
- Here is the ADO release plan work item, which I manually updated to show as "finished" as we are signing off for their Private Preview. https://dev.azure.com/azure-sdk/Release/_workitems/edit/19044.  
- There is no API Spec work item related to the Release Plan and when I open the link to the release plan and then open API readiness milestone, I get an error message stating that there is no associated work item. 
- Here is the link to the Public Preview release plan but it does not direct to a release plan. I see a glimpse of a release plan then get the message that no release plan is found- https://web.powerapps.com/apps/3a732482-e479-4ec1-99c8-359f2c2d3f53?release-plan-id=b097662d-5568-ee11-9ae7-000d3a3418aa
- Also no API Spec work item is found related to the Release Plan for Public Preview. 

I will have the service team create a new Public Preview release plan but wanted to open issue so you could investigate and determine if this is an interim bug introduced as we were adding new functionality and that has now been resolved.

**Comment by maririos** [comment]
> I cannot find any of their release plans when I set the service context to their service - Network Security Perimeter

Could you add a screenshot of the UI you see? I do see the release plans:
<img width="1013" alt="image" src="https://github.com/Azure/azure-sdk-tools/assets/9868623/771c06fe-80ee-4e91-816b-05a71c222990">

> There is no API Spec work item related to the Release Plan 

These release plans got created on 10/11 which is when we still had bug https://github.com/Azure/azure-sdk-tools/issues/7103 this was fixed on 10/12. I can try and re-create that API Spec work item, do you want me to try?

> Here is the link to the Public Preview release plan but it does not direct to a release plan. I see a glimpse of a release plan then get the message that no release plan is found-

The link is wrong :( it is pointing to the DEV instance which is why the Release plan is not found. This was caused by bug https://github.com/Azure/azure-sdk-tools/issues/7104 which has also been fixed. I went and manually update the release plan links in the ADO items so they target the correct instance.

> Also no API Spec work item is found related to the Release Plan for Public Preview.
Same as above. I can try to re-create it now, or as you suggested `abandone` this release plan and having them create a new one.

**Comment by maririos** [comment]
I looked to see if we had other errors for Release Plans with no API work item and, according to the logs from the last 2 weeks, it only happened to this service

**Comment by ladonnaq** [comment]
> I looked to see if we had other errors for Release Plans with no API work item and, according to the logs from the last 2 weeks, it only happened to this service

Okay, not sure what happen but looks like it is resolved now.

**Comment by ladonnaq** [comment]
FYI I can now see the release plans but before I could not.  That may be b/c as you stated the link was going to the wrong environment.  Does the team need to create a new release plan for Public Preview since the SDK release milestone app is not showing up?  Also, perhaps to correct the issue of the missing API Spec work item.

**Comment by maririos** [comment]
> FYI I can now see the release plans but before I could not. That may be b/c as you stated the link was going to the wrong environment. Does the team need to create a new release plan for Public Preview since the SDK release milestone app is not showing up? Also, perhaps to correct the issue of the missing API Spec work item.

Because they haven't done any progress on the Public Preview, I would recommend so. :( apologies for this. 
I can go ahead and mark plan 1125 as Abandoned

**Comment by ladonnaq** [comment]
> > FYI I can now see the release plans but before I could not. That may be b/c as you stated the link was going to the wrong environment. Does the team need to create a new release plan for Public Preview since the SDK release milestone app is not showing up? Also, perhaps to correct the issue of the missing API Spec work item.
> 
> Because they haven't done any progress on the Public Preview, I would recommend so. :( apologies for this. I can go ahead and mark plan 1125 as Abandoned

Yes please mark as Abandoned. I will work with them on creating a new plan for Public Preview. The Public Preview date is not until March 2024.

**Comment by ladonnaq** [comment]
@maririos Once you mark abandoned we can close this issue.

**Comment by maririos** [comment]
Done. Thank you La Donna :)
